0104_MaximumDepthOfBinaryTree.py

```diff
@@ -0,0 +1,47 @@
+class TreeNode(object):
+    def __init__(self, x):
+        self.val = x
+        self.left = None
+        self.right = None
+
+class Solution(object):
+    def maxDepth(self, root):
+        currMaxDepth = 0
+        currRoot = root
+        if currRoot != None:
+            if (self.maxDepth(root.left)>self.maxDepth(root.right)):
+                currMaxDepth = self.maxDepth(root.left)+1
+            else:
+                currMaxDepth = self.maxDepth(root.right)+1
+        return currMaxDepth
+
+
+def arrayToTree(arr, i):
```

Comment:
This function, which is some sort of constructor for TreeNode, should go into the TreeNode class as function ideally constructor.